src/components/Pagination.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Link from 'gatsby-link'
+
+const NavLink = ({ url, text }) => (
+  <Link
+    to={url}
+    activeClassName="font-bold"
+    className={`${
```

Comment:
We have the `classnames` dependency installed, so you'd be able to do something like:

```js
// ...
import cx from 'classnames'

const NavLink = ({ url, text }) => {
  const navLinkClass = cx('no-underline mx-4', {
    'text-grey-lighter pointer-events-none': !url
  })

// ...
className={navLinkClass}
```

You could even use `useMemo` around `classNames`, and pass in `url` as the dependency.